app/templates/about.jade

```diff
@@ -178,178 +162,147 @@ block content
-                  | 15 day minimum vacation policy, excluding company holidays
                 li.small(data-i18n="about.jobs_benefit_3")
-                  | Work from home flexibility
                 li.small(data-i18n="about.jobs_benefit_4")
-                  | Unlimited sick/personal days
                 li.small(data-i18n="about.jobs_benefit_5")
-                  | Professional development and continuing education support
                 li.small(data-i18n="about.jobs_benefit_6")
-                  | Medical/dental/vision insurance
           .col-sm-6.col-md-5.col-lg-4
             .job-listing
-              h5 Software Engineer, iOS
+              h5(data-i18n="about.jobs_positon_1_title")
               .text-center
                 small.label
-                  | San Francisco • Fulltime
-              p.small Want to write the first iPad app for CodeCombat? We’re looking for a product-focused engineer to translate our core gameplay and educational tools into an experience that feels like it was born on the iPad.
+                  | San Francisco •
+                  span.spl(data-i18n="about.fulltime")
+              p.small(data-i18n="about.jobs_positon_1_description")
               a.job-link.btn.btn-lg.btn-navy(href="https://jobs.lever.co/codecombat/eb190007-7195-49ee-a322-893b0d1cdcb4" rel="external")
-                | Learn More
+                span(data-i18n="about.learn_more")
           .col-sm-6.col-md-5.col-md-offset-1.col-lg-4.col-lg-offset-0
             .job-listing
-              h5 Head of Sales
+              h5(data-i18n="about.jobs_positon_2_title")
               .text-center
                 small.label
-                  | San Francisco • Fulltime
-              p.small School districts are scrambling to offer computer science classes to all their students as a core subject. They have had no solution, because they can't afford to hire enough programming teachers – until now. 
+                  | San Francisco •
+                  span.spl(data-i18n="about.fulltime")
+              p.small(data-i18n="about.jobs_positon_2_description")
               a.job-link.btn.btn-lg.btn-navy(href="https://jobs.lever.co/codecombat/2d0194ed-7c69-4aab-b1f9-363ca4b9f04b" rel="external")
-                | Learn More
+                span(data-i18n="about.learn_more")
           .col-sm-6.col-md-5.col-lg-4
             .job-listing
-              h5 Sales Representative
+              h5(data-i18n="about.jobs_positon_3_title")
               .text-center
                 small.label
-                  | San Francisco • Fulltime
-              p.small School districts are scrambling to offer computer science classes to all their students as a core subject. They have had no solution, because they can't afford to hire enough programming teachers – until now. 
+                  | San Francisco •
+                  span.spl(data-i18n="about.fulltime")
+              p.small(data-i18n="about.jobs_positon_3_description")
```

Comment:
Thanks for cleaning up the translations! The job position cards are going to be pulled dynamically from lever in the future, though, and so don't need to be translated. Could you remove just those?